kroxylicious-systemtests/src/main/java/io/kroxylicious/systemtests/steps/KafkaSteps.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests.steps;
+
+import io.kroxylicious.systemtests.Constants;
+import io.kroxylicious.systemtests.resources.manager.ResourceManager;
+import io.kroxylicious.systemtests.templates.strimzi.KafkaTopicTemplates;
+import io.kroxylicious.systemtests.utils.KafkaUtils;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ * The type Kafka steps.
+ */
+public class KafkaSteps {
+    private static final ResourceManager resourceManager = ResourceManager.getInstance();
+
+    /**
+     * Create topic.
+     *
+     * @param clusterName the cluster name
+     * @param topicName the topic name
+     * @param namespace the namespace
+     * @param partitions the partitions
+     * @param replicas the replicas
+     * @param minIsr the min isr
+     */
+    public static void createTopic(String clusterName, String topicName, String namespace,
+                                   int partitions, int replicas, int minIsr) {
+        resourceManager.createResourceWithWait(
```

Comment:
I'm not sure we really need the topic operator.

Reply:
This is the way Strimzi system tests creates the topics, so I just made the same. I didn't think about the operator

Reply:
As it stands, it won't work for any kroxylicious use-case that requires the create topic to be intercepted because the topic operator won't know anything about the kroxylicious. Multi-tenant would be an example.   
For topic encryption, it will work okay.

Reply:
I'll create an issue for that and work on it once this PR is approved if you agree